tests/infra/basicperf.py

```diff
@@ -341,10 +411,24 @@ def cli_args():
         help="Unused, swallowed for compatibility with old args",
         action="store_true",
     )
+    parser.add_argument(
+        "--client-timeout-s",
+        help="Number of seconds after which unresponsive clients are shut down",
+        default=90,
+        type=float,
+    )
+    parser.add_argument(
+        "--rw-mix",
+        help="Run a batched, fractional read/write mix instead of pure writes",
+        type=float,
+    )
 
     return infra.e2e_args.cli_args(parser=parser, accept_unknown=False)
 
 
 if __name__ == "__main__":
     args = cli_args()
-    run(args)
+    if args.rw_mix:
+        run(args, RWMix(1000, 0.5))
```

Comment:
this is probably a silly question but where does the rw-mix cli arg get passed to RWMix?